.github/workflows/check-catalyst.yaml

```diff
@@ -538,7 +539,7 @@ jobs:
             C_COMPILER=$(which gcc) \
             CXX_COMPILER=$(which g++) \
             COMPILER_LAUNCHER="" \
-            ENABLE_LIGHTNING_KOKKOS=ON \
+            # ENABLE_LIGHTNING_KOKKOS=ON \
```

Comment:
The issue is with `clang`, this should be fine here.

```suggestion
            ENABLE_LIGHTNING_KOKKOS=ON \
```

Reply:
It already says gcc just above?

Reply:
yep, that's the reason you don't need to comment this. It's built with gcc/g++ and it should be fine in our CIs

Reply:
Ah I see, thanks!

Reply:
This was reverted since the entire step is skipped.